index.js

```diff
@@ -56,5 +56,37 @@ module.exports = {
         substrings: fieldsReplacements.map(i => i.substring)
       }
     };
+  },
+  /**
+   * Generates a script object which increments fields on the _source document
+   * @param {Object[]} fieldsReplacements Array of objects describing what to replace
+   * @return {{lang: string, source: string, params: *}} Painless Script Object
+   */
+  increment(fieldsMap = []) {
```

Comment:
Maybe the default value for `fieldsMap` should be `{}` ?

Reply:
Absolutely!